Sources/Segnify/View Controllers/PageViewController.swift

```diff
@@ -32,7 +32,18 @@ open class PageViewController: UIViewController {
     // MARK: - Delegates
     
     /// The delegate object of `SegnifyDataSourceProtocol` specifies the content for the `Segnify` instance and this `PageViewController` instance.
-    public private(set) weak var dataSource: SegnifyDataSourceProtocol?
+    public private(set) weak var dataSource: SegnifyDataSourceProtocol? {
+        didSet {
+            let segments = self.pageViewController.view.subviews
+            if let scrollView = segments.first as? UIScrollView {
+                if dataSource?.contentElements.count == 1 {
+                    scrollView.isScrollEnabled = false
+                } else {
+                    scrollView.isScrollEnabled = true
+                }
```

Comment:
How about some space-saving?
```suggestion
                scrollView.isScrollEnabled = (dataSource?.contentElements.count > 1)
```

Reply:
Nice, I felt like it could be simpler, but changed around a lot and couldn't quite find the right one 😅